python/setup.py

```diff
@@ -199,7 +199,7 @@ def _supports_symlinks():
         extras_require={
             'ml': ['numpy>=1.7'],
             'mllib': ['numpy>=1.7'],
-            'sql': ['pandas']
+            'sql': ['pandas>=0.11.0']
```

Comment:
I think it's ok to require 0.13.0, we may need other new Pandas APIs inside pyspark in the future, that only exist after 0.13.0.